### PR TITLE
Rudolf Multiple Shuriken (Ninja Star) Added

### PR DESCRIPTION
**Pull request by bobbyng626**
Create Multiple Object (ninja star and 5 arrow for Henry and Rudolf)
General arrangement for multiple objects creation, for 5 arrows and 5 ninja star for Henry and Rufolf

### CODE REVIEW

File: LF/specialattack.js [hunk 1]
```diff
@@ -101,6 +101,11 @@ function (livingobject, Global, Futil) {
     // Special Attack Projectiles
     1002: function (event, ITR, att, attps, rect) {
       const $ = this
+      if ($.parent.name === 'Rudolf' && $.id === 202) { // bug fix for shuriken not laying on ground
```
Comment (tyt2y3) [inline LF/specialattack.js:104]: What is this bug actually referring to?

Comment (bobbyng626) [reply LF/specialattack.js:104]: As you may see the ninja star ends at the middle in the air.
false_normal_star

The correct demonstration should be as follow, in normal circumstances, the ninja star will disappear after it has dropped on the ground, which the logic is actually applied in the program.
true_normal_star

What I have found is the frame (43 of Rudolf) lost, so I just skipped this frame for now. There maybe some other method but I have just applied the existing one.

Comment (bobbyng626) [reply LF/specialattack.js]: I also edited the character name check to character id check in the next commit

**Comment by tyt2y3** [comment]
Merged. But somehow the distances of them are too small.

**Comment by tyt2y3** [comment]
#27

**Pull request merged**